From 0c10d640c9c13d49c58aa24f2339070983f7cd7f Mon Sep 17 00:00:00 2001
From: moneroexamples <moneroexamples@tuta.io>
Date: Mon, 16 May 2016 05:39:44 +0800
Subject: [PATCH] fix: value-initialization of incomplete type on gcc 6.1

---
 src/crypto/crypto.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/crypto/crypto.cpp b/src/crypto/crypto.cpp
index e47aab0..e120841 100644
--- a/src/crypto/crypto.cpp
+++ b/src/crypto/crypto.cpp
@@ -272,7 +272,7 @@ DISABLE_VS_WARNINGS(4200)
 POP_WARNINGS
 
   static inline size_t rs_comm_size(size_t pubs_count) {
-    return sizeof(rs_comm) + pubs_count * sizeof(rs_comm().ab[0]);
+    return sizeof(rs_comm) + pubs_count * 2 * sizeof(ec_point);
   }
 
   void crypto_ops::generate_ring_signature(const hash &prefix_hash, const key_image &image,
-- 
2.8.2

